ui/dashboards/src/views/ViewDashboard/tests/panelGroups.test.tsx

```diff
@@ -43,6 +43,53 @@ describe('Panel Groups', () => {
     return storeApi;
   };
 
+  beforeEach(() => {
+    const mockIntersectionObserver = jest.fn();
```

Comment:
Oooh, I swear we had to do this at one point (in our internal codebase) and figured out a way to do this globally during test setup (maybe we used a polyfill for it?). We should probably try to figure that out for this package since I'll bet it's going to keep coming up.